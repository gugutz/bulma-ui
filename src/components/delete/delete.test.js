import React from 'react'
import renderer from 'react-test-renderer'
import Delete from './delete'

describe('<Delete /> component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Delete/>).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should support isSmall prop', () => {
    const tree = renderer.create(<Delete isSmall/>).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should support isMedium prop', () => {
    const tree = renderer.create(<Delete isMedium/>).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should support isLarge prop', () => {
    const tree = renderer.create(<Delete isLarge/>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
