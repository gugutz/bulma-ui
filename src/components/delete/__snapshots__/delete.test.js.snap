// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Delete /> component should render correctly 1`] = `
<button
  className="delete"
  type="button"
/>
`;

exports[`<Delete /> component should support isLarge prop 1`] = `
<button
  className="delete is-large"
  type="button"
/>
`;

exports[`<Delete /> component should support isMedium prop 1`] = `
<button
  className="delete is-medium"
  type="button"
/>
`;

exports[`<Delete /> component should support isSmall prop 1`] = `
<button
  className="delete is-small"
  type="button"
/>
`;
